go/vt/vtorc/logic/tablet_discovery.go

```diff
@@ -77,34 +80,50 @@ func updateShardsToWatch() {
 				log.Errorf("Could not parse keyspace/shard %q: %+v", ks, err)
 				continue
 			}
-			newShardsToWatch[k] = append(newShardsToWatch[k], s)
+			if !key.IsValidKeyRange(s) {
+				return fmt.Errorf("Invalid key range %q while parsing clusters to watch", s)
+			}
+			// Parse the shard name into key range value.
+			keyRanges, err := key.ParseShardingSpec(s)
+			if err != nil {
+				return fmt.Errorf("Could not parse shard name %q: %+v", s, err)
```

Comment:
Nit, but errors aren't supposed to use sentence capitalization or punctuation.

Reply:
Done